worlds/dark_souls_3/docs/setup_en.md

```diff
@@ -20,7 +20,7 @@ Get the dinput8.dll from the [Dark Souls III AP Client](https://github.com/Marec
 Then you need to add the two following files at the root folder of your game 
 ( e.g. "SteamLibrary\steamapps\common\DARK SOULS III\Game" ):  
 - **dinput8.dll**
-- **AP.json** (renamed from the generated file AP-{ROOM_ID}.json)
+- **AP.json** (this is the .json file provided in the multiworld room or by the host, formatted as AP-{ROOM_ID}.json.  Rename this to AP.json)
```

Comment:
```suggestion
- **AP.json**: the download provided in the multiworld room or by the host, named AP-{ROOM_ID}.json, has to be randed to AP.json.
```
maybe? not a native speaker myself, but i think having the actual instructions before the '.' makes it harder to skip it when reading.

Reply:
You're right, I rephrased the sentence by mixing the two suggestions